app/javascript/app/components/sectors-agriculture/countries-context/meat-data/meat-data-component.jsx

```diff
@@ -0,0 +1,141 @@
+import infoIcon from 'assets/icons/info';
+import Tooltip from './tooltip';
+
+import styles from './meat-data-styles.scss';
+
+const cardTheme = {
+  card: styles.card,
+  data: styles.data
+};
+
+const MeatData = ({
+  selectedCountry,
+  selectedYear,
+  categories,
+  breakByOptions,
+  selectedCategory,
+  selectedBreakBy,
+  chartData,
+  chartConfig,
+  dataSelected,
+  dataOptions,
+  domain,
+  updateCategoryFilter,
+  updateBreakByFilter,
+  handleLegendChange,
+  handleInfoBtnClick
+}) => (
+  <div className={styles.container}>
+    {chartData && (
+      <Card
+        key={'Production, Consumption and Trade'}
+        title={'Production, Consumption and Trade'}
+        theme={cardTheme}
+      >
+        <div className={styles.cardContainer}>
+          <div className={styles.header}>
+            <div className={styles.title}>
+              Many crops grown within a country are traded globally and consumed
+              elsewhere. Explore countries’ production, consumption and trade of
+              some of the main agricultural products.
+            </div>
+            <Icon
+              icon={infoIcon}
+              theme={{ icon: styles.cardInfoIcon }}
+              onClick={handleInfoBtnClick}
+            />
+          </div>
+          <div className={styles.actionsContainer}>
+            {categories && (
+              <Dropdown
+                label={'Categories'}
+                value={selectedCategory}
+                options={categories}
+                onValueChange={updateCategoryFilter}
+                hideResetButton
+              />
+            )}
+            {breakByOptions && (
+              <Dropdown
+                label={'Break by'}
+                value={selectedBreakBy}
+                options={breakByOptions}
+                onValueChange={updateBreakByFilter}
+                hideResetButton
+              />
+            )}
+          </div>
+          <div className={styles.chartContainer}>
+            {chartData && chartData.hasValues ? (
+              <Chart
+                type="bar"
+                theme={{ legend: styles.legend }}
+                config={chartConfig}
+                data={chartData.data}
+                domain={domain}
+                customTooltip={<Tooltip />}
+                dataSelected={dataSelected}
+                dataOptions={dataOptions}
+                height={300}
+                onLegendChange={handleLegendChange}
+                barSize={40}
+                barGap={0}
+              />
+            ) : (
+              <NoContent
+                message="No data for selected filters"
+                className={styles.noContent}
+                minHeight={300}
+              />
+            )}
+          </div>
+        </div>
+        <div className={styles.yearData}>
+          <span>{selectedYear && selectedYear.value}</span> data
+        </div>
+      </Card>
+    )}
+    {selectedCountry &&
```

Comment:
I think you can save params as a one object (before `return (...)`):
`const params = selectedCountry && selectedYear && { country: selectedCountry.value, year: selectedYear.value };`
and then use `params` object as a guard here:
`{params && [ <MeatConsumptionProvider... ]}`